src/siembol/resources/git.config

```diff
@@ -0,0 +1,5 @@
+[user]
```

Comment:
i think this is not needed if people trying from their github accounts

Reply:
ok -- I thought this was a resource that had to exist, for some reason.  

If it's not necessary, great.  One fewer thing for the user to customize (and I completely forgot to document this step, anyway, so better yet if I can just remove it :D)